cmd/config.go

```diff
@@ -120,17 +116,45 @@ func setNodeCmdRun(cmd *cobra.Command, args []string) error {
 	}
 }
 
-func importCmdRun(cmd *cobra.Command, args []string) error {
-	vms, err := utils.VirtualMachineScaleSetVMsViaKubeconfig()
-	if err != nil {
-		return fmt.Errorf("failed to get VMSS VMs: %w", err)
+func importCmdCommand() *cobra.Command {
+	var subscriptionID string
+	var resourceGroup string
+	var clusterName string
+
+	cmd := &cobra.Command{
+		Use:          "import",
+		Short:        "Import Kubernetes nodes in the configuration",
+		SilenceUsage: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			utils.DefaultSpinner.Start()
```

Comment:
Nice feature. We could add it to the other commands to avoid printing "Running..." and make the output cleaner.

Reply:
https://github.com/Azure/kubectl-aks/pull/40